README.md

```diff
@@ -51,7 +51,7 @@ files in place.
 ```sh
 make lint-fix
 
-#make lint-fix-go # Go linter does not support fixes
+make lint-fix-go # only lint and fix Go files
```

Comment:
Noticed now that this says `lint-fix-go` and `lint-fix-md` while the Makefile says `lint-go-fix` and `lint-md-fix` respectively.

Reply:
The same issue seems to be present int the other repos as well.

Reply:
So sorry for the email spam, but it should be resolved now.